tests/experimental/datasources/integration_test_util.py

```diff
@@ -0,0 +1,131 @@
+from great_expectations import DataContext
+from great_expectations.checkpoint import SimpleCheckpoint
+from great_expectations.data_context.types.base import IncludeRenderedContentConfig
+from great_expectations.experimental.datasources.interfaces import BatchRequest
+from great_expectations.render import (
+    AtomicDiagnosticRendererType,
+    AtomicPrescriptiveRendererType,
+)
+
+
+def run_checkpoint_and_datadoc_on_taxi_data_2019_01(
```

Comment:
This was basically pulled from `test_integration_sql_datasource.py::test_run_checkpoint_and_data_doc`. If we use the same underlying data for different datasource, we should get the same computed expectations.

Reply:
I try to avoid using helpers in tests whenever possible.
IMO it becomes too easy to pull everything into helpers that obfuscate what's actually being tested.
Although I think you could argue pytest fixtures are basically the same thing. The `Arrange` phase works very naturally with fixtures.

Is there any part of this that would be useful to extract into a parametrized fixture that yields the necessary pandas, SQL and postgres pieces?
https://docs.pytest.org/en/6.2.x/fixture.html#parametrizing-fixtures


I do think it would be easier to go down a rabbit hole here and spend way too much time on it.
I'm fine to push some or all of this work to a followup (or even not do it at all).

Reply:
Done. I moved everything to `test_integration_datasource.py` and the tests are now parameterized by the datasource type. This now more clearly highlights what is under test.

There is still a helper function `_configure_and_run_data_assistant` that contains boiler plate code for running the data assistant. Folding that into a fixture would make what's under test less clear I believe.